py/server/deephaven/experimental/ema.py

```diff
@@ -0,0 +1,106 @@
+#
+#     Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+"""This modules allows users to define exponential moving averages that can be used in the Deephaven query language."""
+import numpy
+import jpy
+
+_JAbstractMAType = jpy.get_type('io.deephaven.numerics.movingaverages.AbstractMa$Type')
+_JAbstractMATMode = jpy.get_type('io.deephaven.numerics.movingaverages.AbstractMa$Mode')
+_JBadDataBehavior = jpy.get_type('io.deephaven.numerics.movingaverages.ByEma$BadDataBehavior')
+_JTimeUnit = jpy.get_type('java.util.concurrent.TimeUnit')
+_JByEmaSimple = jpy.get_type('io.deephaven.numerics.movingaverages.ByEmaSimple')
+_JEma = jpy.get_type('io.deephaven.numerics.movingaverages.Ema')
+_JEmaArray = jpy.get_type('io.deephaven.numerics.movingaverages.EmaArray')
+_JExponentiallyDecayedSum = jpy.get_type('io.deephaven.numerics.movingaverages.ExponentiallyDecayedSum')
+
+
+def _convertEnumBehavior(value, enumType):
+    if value is None or isinstance(value, enumType):
+        return value
+    elif hasattr(enumType, value):
+        return getattr(enumType, value, None)
+    return value
+
+
+def ByEmaSimple(nullBehavior, nanBehavior, mode, timeScale, timeUnit, type=None):
+    """Constructor for an engine aware Exponential Moving Average (EMA) which performs a `groupBy` ema calculation without
```

Comment:
```suggestion
    """Constructor for an engine aware Exponential Moving Average (EMA) which performs a `group_by` ema calculation without
```

Reply:
I think groupBy is OK because we are talking about the operation on the server.